WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -828,19 +828,56 @@ def create
   end
 
   def create_paypal_order
-    return head :forbidden if Rails.env.production?
+    return head :forbidden if EnvConfig.WCA_LIVE_SITE?
 
     @registration = registration_from_params
-    render json: PaypalInterface.create_order(@registration)
+    render json: PaypalInterface.create_order(@registration, params[:total_charge], params[:currency_code])
```

Comment:
I feel like the `currency_code` parameter might be redundant, as it can be safely derived from the `registration` object and it won't change even when a donation is being made.

(The `total_charge` parameter is justified because of donations, i.e. when a volunteer donation is being made the actual PayPal charge can differ from the amount affiliated with the registration. But for currencies, this is not the case.)